save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt

```diff
@@ -6,83 +6,130 @@
 
 package com.saveourtool.save.frontend.components.views.fossgraph
 
-import com.saveourtool.save.frontend.externals.fontawesome.faSearch
+import com.saveourtool.save.entities.vulnerability.VulnerabilityDto
+import com.saveourtool.save.entities.vulnerability.VulnerabilityProjectDto
+import com.saveourtool.save.frontend.components.tables.TableProps
+import com.saveourtool.save.frontend.components.tables.columns
+import com.saveourtool.save.frontend.components.tables.tableComponent
+import com.saveourtool.save.frontend.components.tables.value
 import com.saveourtool.save.frontend.externals.progressbar.Color
 import com.saveourtool.save.frontend.externals.progressbar.progressBar
-import com.saveourtool.save.frontend.utils.Style
-import com.saveourtool.save.frontend.utils.buttonBuilder
-import com.saveourtool.save.frontend.utils.useBackground
+import com.saveourtool.save.frontend.utils.*
 
 import csstype.AlignItems
 import csstype.ClassName
 import csstype.Display
 import js.core.get
 import js.core.jso
 import react.*
-import react.dom.aria.ariaDescribedBy
-import react.dom.aria.ariaLabel
+import react.dom.html.ReactHTML
 import react.dom.html.ReactHTML.div
-import react.dom.html.ReactHTML.form
 import react.dom.html.ReactHTML.h1
 import react.dom.html.ReactHTML.h6
-import react.dom.html.ReactHTML.input
-import react.dom.html.ReactHTML.span
 import react.dom.html.ReactHTML.textarea
+import react.router.dom.Link
 import react.router.useParams
-import web.html.InputType
 
 /**
  * [VFC] for foss graph view
  */
 @Suppress("MAGIC_NUMBER")
 val fossGraphView: VFC = VFC {
+    val params = useParams()
+    val vulnerabilityName = params["vulnerabilityName"]!!.toString()
+
     fossGraph {
-        name = "CVE-2022-22978"
-        progress = 87
+        name = vulnerabilityName
     }
 }
 
 @Suppress(
     "MAGIC_NUMBER",
     "TOO_LONG_FUNCTION",
     "LongMethod",
+    "TYPE_ALIAS",
 )
 val fossGraph: FC<FossGraphViewProps> = FC { props ->
     useBackground(Style.WHITE)
 
     val params = useParams()
     val vulnerabilityName = params["vulnerabilityName"]!!.toString()
 
-    div {
-        className = ClassName("card card-body mt-0")
+    val (vulnerability, setVulnerability) = useState<VulnerabilityDto?>(null)
```

Comment:
I do not like an idea of `vulnerability` being nullable. Maybe some stub?

Reply:
Ok. Added _empty_